### PR TITLE
Scheduler-UI-NEXT from develop to master Code push regarding CI Build Configurations and some fixes

### PR DESCRIPTION
**Pull request by bhpratibha**
Scheduler-UI-NEXT from develop to master Code push regarding CI Build Configurations and some fixes

### CODE REVIEW

File: src/environments/environment.ci.ts.template [hunk 1]
```diff
@@ -29,7 +29,7 @@ const COMMON_API_OPEN = '<%= COMMON_API_BASE %>';
 const COMMON_API = '<%= COMMON_API_BASE %>';
 const SCHEDULER_API =  '<%= SCHEDULER_API_BASE %>';
 const TM_API = '<%= TM_API_BASE %>';
-
+const MMU_API = `http://${IP}:8080/mmuapi-v1.0/`;
```
Comment (drtechie) [inline src/environments/environment.ci.ts.template]: This doesn't look right.
http and port numbers shouldn't be here.

Comment (drtechie) [reply src/environments/environment.ci.ts.template]: Even the version of the mmu API

Comment (bhpratibha) [reply src/environments/environment.ci.ts.template]: @drtechie HI Mithun, updated as per comments, Please check

**Review by helenKaryamsetty — APPROVED** [review]
looks fine

**Pull request merged**